src/current/_includes/releases/v24.2/v24.2.8.md

```diff
@@ -0,0 +1,87 @@
+
+<h3 id="v24-2-8-sql-language-changes">SQL language changes</h3>
+
+- Added the `legacy_varchar_typing` session setting, which reverts the changes of [#133037](https://github.com/cockroachdb/cockroach/pull/133037) that causes the change in typing behavior described in [#137837](https://github.com/cockroachdb/cockroach/pull/137837). Specifically, it makes type-checking and overload resolution ignore the newly added "unpreferred" overloads. This setting defaults to `on`. [#137920][#137920]
+
+<h3 id="v24-2-8-operational-changes">Operational changes</h3>
+
+- Removed the `sql.auth.resolve_membership_single_scan.enabled` cluster setting. This was added out of precaution in case it was necessary to revert back to the old behavior for looking up role memberships, but this escape hatch has never been needed in practice since this was added in v23.1. [#136161][#136161]
+- Telemetry delivery is now considered successful even in cases where we experience a network timeout. This will prevent throttling in cases outside an operator's control. [#136479][#136479]
+- When a schema change job is completed, rolls back, or encounters a failure, the time taken since the job began is now logged in a structured log in the `SQL_SCHEMA` log channel. [#136928][#136928]
+- Added a new configurable parameter `kv.transaction.max_intents_and_locks` that will prevent transactions from creating too many intents. [#137700][#137700]
+- Added the metric `txn.count_limit_rejected`, which tracks the KV transactions that have been aborted because they exceeded the max number of writes and locking reads allowed. [#137700][#137700]
+- Added the metric `txn.count_limit_on_response`, which tracks the number of KV transactions that have exceeded the count limit on a response. [#137700][#137700]
+
+<h3 id="v24-2-8-bug-fixes">Bug fixes</h3>
+
+- `ALTER COLUMN SET NOT NULL` was not enforced consistently when the table was created in the same transaction. [#136365][#136365]
+- Fixed a bug where `CREATE RELATION / TYPE` could leave dangling namespace entries if the schema was concurrently being dropped. [#136379][#136379]
+- The `idle_in_session_timeout` setting now excludes the time spent waiting for schema changer jobs to complete, preventing unintended session termination during schema change operations. [#136502][#136502]
+- Fixed a bug that causes the optimizer to use stale table statistics after altering an `ENUM` type used in the table. [#136759][#136759]
+- Table statistics collection in CockroachDB could previously run into `no bytes in account to release` errors in some edge cases (when the SQL memory budget, configured via `--max-sql-memory` flag, was close to being exhausted). The bug has been present since v21.2 and is now fixed. [#136165][#136165]
+- CockroachDB now better respects `statement_timeout` limit on queries involving the top K sort and merge join operations. [#136652][#136652]
```

Comment:
nit: "the" statement timeout limit ? 

PS i'm only looking so close because i'm copying all your edits over to v24.1.10 line by line :-)

Reply:
Done!